src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -187,4 +218,13 @@ private void initWithPaginationToken(PaginationToken token, int unfilteredCount,
             })
         .collect(Collectors.toList());
   }
+
+  private Map<String, String> defineAcceptableSortFields() {
+    return Map.of(
+      "projectTitle", "projectTitle",
+      "researcher", "researcher",
+      "darCode", "dar_code",
+      "institution", "institution_name"
+    );
+  }
```

Comment:
DRY. Related to my comment in the resource class, if you find yourself writing the same code twice, consider moving it to a more useful location. I could envision a custom interface for certain model classes where any class that wanted to have custom sort orders and sort fields would be forced to implement this method. Definitely not necessary at this point, but good to keep in mind.